endpoint/app.js

```diff
@@ -28,7 +30,7 @@ app.all('*',(req,response,next) => {
 
 app.use((err, req, res, next) => {
     if (err.code === '23503') {
-      res.status(400).send({ msg: 'foreign key violation' });
+      res.status(404).send({ msg: 'foreign key violation' });
```

Comment:
I would look to not expose too much information about your underlying database for security reasons